tests/common.go

```diff
@@ -1356,6 +1366,50 @@ func GetAppNamespace(ctx *scheduler.Context, taskname string) string {
 	return ctx.App.GetID(fmt.Sprintf("%s-%s", taskname, Inst().InstanceID))
 }
 
+// Using the cuurent Context (kubeconfig) to generate schedule options
+// NOTE: When using a ScheduleOption that was created during a context (kubeconfig)
+// that is different from the current context, make sure to re-generate ScheduleOptions
+func CreateScheduleOptions(errChan ...*chan error) scheduler.ScheduleOptions {
+	var err error
+	options := scheduler.ScheduleOptions{
```

Comment:
To improve readability, it might be helpful to create two separate ScheduleOptions variables depending on the cluster configuration, instead of updating the same variable in both the if and else blocks.

Reply:
How about adding a docstring that describes what the function does?

Reply:
While the function wasn't written by me (I just extracted it from another function and put it in here), I have made the changes you suggested.

Reply:
docstring already exists & updated

Reply:
✅